From 70d236dedadc030fd2b450d7607b395b50523538 Mon Sep 17 00:00:00 2001
From: Mark Reynolds <mreynolds@redhat.com>
Date: Thu, 19 Oct 2017 17:02:20 -0400
Subject: [PATCH] Ticket 49407 - status-dirsrv shows ellipsed lines

Bug Description:  To show the full output you have to pass "-l" to systemctl,
                  but there is no way to use this option with the current design.

Fix Description:  Just show the full lines by default, as adding options can break
                  the script's current usage.

https://pagure.io/389-ds-base/issue/49407

Reviewed by: tbordaz(Thanks!)

(cherry picked from commit 45d2fd4b50227687ad042a0e17d8dcd9e4cd3023)
---
 ldap/admin/src/scripts/status-dirsrv.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/ldap/admin/src/scripts/status-dirsrv.in b/ldap/admin/src/scripts/status-dirsrv.in
index 90428990b..8e492c115 100755
--- a/ldap/admin/src/scripts/status-dirsrv.in
+++ b/ldap/admin/src/scripts/status-dirsrv.in
@@ -37,7 +37,7 @@ status_instance() {
     # Use systemctl if available.
     #
     if [ -d "@systemdsystemunitdir@" ] && [ $(id -u) -eq 0 ];then
-        @bindir@/systemctl status @package_name@@$SERV_ID.service
+        @bindir@/systemctl status @package_name@@$SERV_ID.service -l
         rv=$?
         if [ $rv -ne 0 ]; then
             return 1
@@ -65,7 +65,7 @@ found=0
 if [ $# -eq 0 ]; then
     # We're reporting the status of all instances.
     ret=0
-    @bindir@/systemctl status @package_name@.target
+    @bindir@/systemctl status @package_name@.target -l
     initfiles=`get_initconfig_files $initconfig_dir` || { echo No instances found in $initconfig_dir ; exit 1 ; }
     for i in $initfiles; do
         inst=`normalize_server_id $i`
-- 
2.13.6

